.env.staging.example

```
@@ -0,0 +1,63 @@
+APP_NAME="Toro GYM Staging"
+APP_ENV=staging
+APP_KEY=
+APP_DEBUG=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_DEBUG key should go before the APP_ENV key

(UnorderedKey)